client/example_test.go

```diff
@@ -218,3 +218,46 @@ func ExampleClient_Store() {
 
 	log.Println("Message has been marked as seen")
 }
+
+func ExampleClient_Search() {
+	// Let's assume c is a client
+	var c *client.Client
+
+	// Select INBOX
+	_, err := c.Select("INBOX", false)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Set search criteria
+	criteria := imap.NewSearchCriteria()
+	criteria.WithoutFlags = []string{imap.SeenFlag}
+	ids, err := c.Search(criteria)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println("IDs found:", ids)
+
+	if len(ids) > 0 {
+
```

Comment:
Style: extraneous line here

Reply:
oh yes I'll change that. Weird thing gofmt didn't complain about that.